ui/perfherder/graphs/GraphsView.jsx

```diff
@@ -166,6 +177,15 @@ class GraphsView extends React.Component {
     }
   };
 
+  getChangelogData = async () => {
+    const rawData = await getData(createApiUrl(endpoints.changelog));
+    const changelogData = rawData.data.map(({ date, ...extra }) => ({
+      date: new Date(date),
+      ...extra,
+    }));
+    this.setState({ changelogData });
+  };
+
```

Comment:
I'm curious why you'd fetch the changelog data here when it's not being used in this component - it's only passed as a prop to `GraphsViewControls`?